### PR TITLE
Made a TUI for the app

### LINKED ISSUE #3: Rework the interface

### PR DESCRIPTION
**Pull request by Minemobs**
closes #3

### CODE REVIEW

File: src/ui.rs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    let block = Block::default()
+        .title(" Roumanji (latin char) ")
+        .title_alignment(Alignment::Center)
+        .borders(Borders::ALL);
+    let rect = Rect::new(
+        h_chunk[1].x - 10,
+        chunks[3].y,
+        h_chunk[1].width + 20,
+        chunks[3].height,
+    );
```
Comment (Minemobs) [inline src/ui.rs:61]: Those `Rect::new` are only a temporary fix until I find a good way of positioning items

Comment (noaottermann) [reply src/ui.rs:61]: why -10 or +20

Comment (Minemobs) [reply src/ui.rs:61]: I just played with a bunch of random number until I found one that made the UI look "nice"

**Comment by Minemobs** [comment]
I believe that the UI is good enough to be merged into main

**Pull request merged**